mate-academy/layout_landing-page
# Develop

The Met website learning project.
css, html, no JS

Preview link wont work because of style file path in index html.

instructions on how to preview file locally:

- clone project to your pc
- run "npm i" within project folder
- switch to develop branch
- run "npm start" within project folder
- check site on http://localhost:1234

- [preview link] (https://taniakolesnik.github.io/layout_landing-page/)
- [figma project](https://www.figma.com/file/lSR1m42L9YwzQwzzxKwHpw/THE-MET)

PS: god/gods help you checking this ugly code
